robot_smach_states/src/robot_smach_states/perception.py

```diff
@@ -31,10 +43,10 @@ def run(self, robot, entity, keep_following, waittime):
         # That would be equivalent to defining coordinates 0,0,0 in its own frame, so that is what we do here.
         # The added benefit is that the entity's frame actually moves because the entity is tracked.
         # This makes the head track the entity
-        vs = VectorStamped.from_xyz(0, 0, 0, entity.last_update_time, frame_id=entity.uuid)
-        rospy.loginfo('Look at %s' % (repr(vs)))
-        robot.head.look_at_point(vs)
-        rospy.sleep(rospy.Duration(waittime))
+        vs = VectorStamped.from_xyz(0, 0, height, entity.last_update_time, frame_id=entity.uuid)
+        rospy.loginfo(f'Look at {vs}')
+        self._robot.head.look_at_point(vs)
+        rospy.sleep(rospy.Duration(self._waittime))
```

Comment:
what an ugly thing...
But it was here beforehand so no dealbreaker.

Reply:
`self._waittime` should be stored as a `rospy.Duration` in the constructor